app/controllers/groups_controller.rb

```diff
@@ -3,7 +3,15 @@ class GroupsController < ApplicationController
   before_action :require_leader, only: [:edit, :update]
 
   def index
+    active_groups = Group.order(:name).select{ |g| !g.inactive? }
+    active_groups = GroupDecorator.decorate_collection(active_groups)
+    @groups = Kaminari.paginate_array(active_groups)
+      .page(params[:page]).per(params[:per])
+  end
+
+  def all
```

Comment:
Miert szedjuk kulon?
Szerintem szebb az ha siman csak aktivitas alapjan sorrendezzuk. Esetleg maskent jelenitsuk meg az aktivakat az inaktivtol. 
De ha mindenkepp kulon akarjuk, szerintem akkor is maradhatna az indexben. Es a view-ba raknek egy kapcsolot amivel az osszeset vagy csak az aktivakat mutatjuk.

Reply:
With separation by url you can easily preserve the previous state of the page, if users want to link it or use browsers back button.

Reply:
In that case, we use query string in url sth like '/?show=all' or '?show=active?'

Reply:
I don't entirely agree with this. Query strings are meant for passing variables that can take any values (ex. search query, or page count). This is not the case here, so even if one route handles both options the route could be something like `/show/all` and `/show/active`.

And yea, I know I'm a bit late with this :smile: